src/Common/CodeAnalysis.ruleset

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0"?>
```

Comment:
rulesets are actually deprecated from Visual Studio 16.3+, the way to control diagnostic severities is now exclusively via editorconfig. in fact there is no way to create a ruleset anymore in VS and using the "suppress" feature in VS automatically updates editorconfig too

you can move it all into editorconfig using the `dotnet_diagnostic. SA1122.severity = none` syntax
I believe that would be much nice and cleaner

Reply:
I forgot to attach the relevant links:
 - https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019#convert-an-existing-ruleset-file-to-editorconfig-file `Starting in Visual Studio 2019 version 16.5, ruleset files are deprecated in favor of the EditorConfig file for analyzer configuration for managed code (...)`
 - https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019#set-rule-severity-in-the-rule-set-file (see the "note" box)

Reply:
Yes, this seems better, thanks. :)